packages/lesswrong/components/languageModels/LanguageModelChat.tsx

```diff
@@ -362,16 +363,21 @@ export const LanguageModelChat = ({classes}: {
   classes: ClassesType<typeof styles>,
 }) => {
   return <DeferRender ssr={false}>
-    <div className={classes.root}>
-      <ChatInterface classes={classes} />
-    </div>
+    <AnalyticsContext pageSectionContext='llmChat'>
+      <div className={classes.root}>
+        <ChatInterface classes={classes} />
+      </div>
+    </AnalyticsContext>
   </DeferRender>;
 }
 
 const LanguageModelChatComponent = registerComponent('LanguageModelChat', LanguageModelChat, {styles});
 
+const LlmChatMessageComponent = registerComponent('LlmChatMessage', LLMChatMessage, {styles});
```

Comment:
My guess is that you'll need to no longer pass `classes` in to `LlmChatMessage` as a prop, and am surprised it's not complaining about it

Reply:
Nope, it breaks without this